### PR TITLE
DOCS-6712-fixed doc bugs

### PR DESCRIPTION
**Pull request by priyaayyar**

### CODE REVIEW

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc [hunk 1]
```diff
@@ -6,7 +6,12 @@ endif::[]
 
 image::prepare-to-install-breadcrumb.png[]
 
-Before you download and install Anypoint Service Mesh, you must meet the  software, licensing, and permissions and roles requirements describes in this section.
+Before you download and install Anypoint Service Mesh, make sure that you review and fulfill the software, licensing, and permissions and roles requirements describes in this section. 
+
+== About the Audience
+
+Anypoint Service Mesh is a enterprise microservices management application. Microservices are deployed in containerized environments to achieve best business results. Anypoint Service Mesh is developed using Istio and Kubernetes. Therefore, you must have basic working knowledge of Istio and Kubernetes to be able to install and use (derive benefit from??) Anypoint Service Mesh.
```
Comment (tbruno) [inline modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: >(derive benefit from??)

We need to remove this?

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc [hunk 2]
```diff
@@ -40,7 +50,7 @@ You require a Anypoint Service Mesh license to perform the installation. If you
 
 Before you can use Anypoint Service Mesh, you must have the following roles and permissions:
  
-* API Manager Environment Administrator permission or Organization Administrator role for your organization, to access the client ID and client secret credentials.
+* API Manager Environment Administrator or Organization Administrator permission for your organization, to access the client ID and client secret credentials.
```
Comment (tbruno) [inline modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: We need to revert this change. One thing is a permission, another one a role.

File: modules/ROOT/pages/provision-adapter-configure-service-mesh.adoc [hunk 3]
```diff
@@ -27,6 +27,8 @@ You can also modify the size of an adapter after it has been provisioned. If you
 
 After the new adapter is ready, the old pod is terminated and all requests are routed to the new adapter. If the update fails to start up a new pod due to insufficient space, or if the pod is not ready, an error message is recorded in a Kubernetes event. In such a scenario, the old adapter continues to handle requests because the new pod is not yet available.
 
+You can delete an adapter if it is no longer required or if you want to troubleshoot an issue. However, you must delete all existing bindings for the adapter before you delete the adapter itself.
```
Comment (tbruno) [inline modules/ROOT/pages/provision-adapter-configure-service-mesh.adoc]: I wouldn't put `or if you want to troubleshoot an issue`

**Review by tbruno — COMMENTED** [review]
Thanks for starting applying the feedback. Small comments

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc (hunk 1 above)
Comment (chiragnarangmulesoft) [reply modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: I would just keep this much 
"Anypoint Service Mesh is developed using Istio and Kubernetes. Therefore, you must have a basic working knowledge of Istio and Kubernetes to be able to install and use Anypoint Service Mesh"

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc [hunk 4]
```diff
@@ -24,11 +29,16 @@ For more information about how to install Helm, see the Helm installation docume
...
 * Install Kubernetes 1.12+ with admissionregistration API 
-*Tip:* To verify the installation, use the command `kubectl api-versions | grep admissionregistration.k8s.io/v1beta1`.
++
+*Tip:* To verify the installation, use the command: 
++
+`kubectl api-versions | grep admissionregistration.k8s.io/v1beta1`.
 * Configure kube-apiserver process with the admission-control flag to “MutatingAdmissionWebhook”
-For more information about how to turn on, see tke Kubernetes documentation.
++
+For more information about how to turn on *turn on what--Tomas?*, see the Kubernetes documentation.
```
Comment (chiragnarangmulesoft) [inline modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: Incomplete sentence "For more information about how to turn on *turn on what--Tomas?*"

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc (hunk 1 above)
Comment (chiragnarangmulesoft) [reply modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: I would just put this much "you must have a basic working knowledge of Istio and Kubernetes to be able to install and use Anypoint Service Mesh"

**Review by chiragnarangmulesoft — CHANGES_REQUESTED** [review]
Added some minor comments. Please review & merge

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc (hunk 1 above)
Comment (priyaayyar) [reply modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: Done! Thanks!

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc (hunk 2 above)
Comment (priyaayyar) [reply modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: Done! Thanks!

File: modules/ROOT/pages/provision-adapter-configure-service-mesh.adoc (hunk 3 above)
Comment (priyaayyar) [reply modules/ROOT/pages/provision-adapter-configure-service-mesh.adoc]: Done! Thanks!

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc (hunk 4 above)
Comment (priyaayyar) [reply modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: Exactly my point. I'll remove it for now.

File: modules/ROOT/pages/bind-api-configure-service-mesh.adoc [hunk 5]
```diff
@@ -4,11 +4,9 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: service mesh, microservices, binding
 
-When you create the binding, Anypoint Service Mesh creates a three-way association of the service in Kubernetes with the API that you created in Anypoint Platform to represent the service, and the adapter that you created for the service. The adapter uses the unique API ID of the API to fetch policies and apply them to your services that are running in the cluster.
+When you create the binding, Anypoint Service Mesh creates a three-way association of the service in Kubernetes with the API that you created in Anypoint Platform to represent the service, and the adapter that you created for the service. The adapter uses the unique API ID to fetch policies and apply them to your services that are running in the cluster.
```
Comment (kbostrom) [inline modules/ROOT/pages/bind-api-configure-service-mesh.adoc]:
```suggestion
When you create the binding, Anypoint Service Mesh creates a three-way association of the service in Kubernetes, the API that you created in Anypoint Platform to represent the service, and the adapter that you created for the service. The adapter uses the unique API ID to fetch policies and apply them to your services that are running in the cluster.
```

File: modules/ROOT/pages/bind-api-configure-service-mesh.adoc [hunk 6]
```diff
@@ -21,7 +19,7 @@ In the following example, a binding called “customers” is created to bind th
 NAMESPACE      NAME        ADAPTER                         API ID  SERVICE NAME    STATUS
 nto-payments   customers   anypoint-service-mesh-adapter   1234    customers       Ready
 ----
-
+[[bind_the_service]]
 == Binding the service
```
Comment (kbostrom) [inline modules/ROOT/pages/bind-api-configure-service-mesh.adoc]:
```suggestion
== Binding the Service
```

File: modules/ROOT/pages/manage-apis-with-service-mesh.adoc [hunk 7]
```diff
@@ -6,10 +6,14 @@ endif::[]
 
 image::manage-service-mesh-breadcrumb.png[]
 
-After you configure Anypoint Service Mesh, you can manage both Mule and non-Mule applications from Anypoint API Manager, including applying policies to secure your microservices.
+After you configure Anypoint Service Mesh, you can manage both Mule and non-Mule applications from Anypoint API Manager, including applying policies to secure your microservices. 
+
+Not all policies available in Anypoint Platform are applicable to Anypoint Service Mesh. Only a subset of those policies apply. Additionally, the configurations within each of the policies that are applicable to Anypoint Service Mesh also differ as compared with the policies that are applicable to Anypoint Platform.
```
Comment (kbostrom) [inline modules/ROOT/pages/manage-apis-with-service-mesh.adoc]:
```suggestion
Not all policies available in Anypoint Platform are applicable to Anypoint Service Mesh. Only a subset of those policies apply. Additionally, the configurations within each of the policies that are applicable to Anypoint Service Mesh also differ from the policies that are applicable to Anypoint Platform.
```

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc [hunk 8]
```diff
@@ -6,7 +6,12 @@ endif::[]
 
 image::prepare-to-install-breadcrumb.png[]
 
-Before you download and install Anypoint Service Mesh, you must meet the  software, licensing, and permissions and roles requirements describes in this section.
+Before you download and install Anypoint Service Mesh, make sure that you review and fulfill the software, licensing, and permissions and roles requirements describes in this section.
```
Comment (kbostrom) [inline modules/ROOT/pages/prepare-to-install-service-mesh.adoc]:
```suggestion
Before you download and install Anypoint Service Mesh, ensure that you review and fulfill the software, licensing, and permissions and roles requirements described in this section. 
```

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc [hunk 9]
```diff
@@ -6,7 +6,12 @@ endif::[]
 
 image::prepare-to-install-breadcrumb.png[]
 
-Before you download and install Anypoint Service Mesh, you must meet the  software, licensing, and permissions and roles requirements describes in this section.
+Before you download and install Anypoint Service Mesh, make sure that you review and fulfill the software, licensing, and permissions and roles requirements describes in this section. 
+
+== About the Audience
```
Comment (kbostrom) [inline modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: Consider "Audience Requirements," for parallelism and also specificity.

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc [hunk 10]
```diff
@@ -24,11 +29,14 @@ For more information about how to install Helm, see the Helm installation docume
...
 
-If you want to enable automatic sidecar injection, you must additionally have:
+If you want to enable automatic sidecar injection, you must additionally:
+
 * Install Kubernetes 1.12+ with admissionregistration API 
-*Tip:* To verify the installation, use the command `kubectl api-versions | grep admissionregistration.k8s.io/v1beta1`.
++
+*Tip:* To verify the installation, use the command: 
++
+`kubectl api-versions | grep admissionregistration.k8s.io/v1beta1`.
```
Comment (kbostrom) [inline modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: Consider making this a bullet item parallel to Install... and Add and list:

* Install Kubernetes...
* Verify the installation using...
* Configure kube-apiserver...
* Add and list

**Review by kbostrom — APPROVED** [review]
Voila!

File: modules/ROOT/pages/bind-api-configure-service-mesh.adoc (hunk 5 above)
Comment (priyaayyar) [reply modules/ROOT/pages/bind-api-configure-service-mesh.adoc]: Done! Thanks!

File: modules/ROOT/pages/bind-api-configure-service-mesh.adoc (hunk 6 above)
Comment (priyaayyar) [reply modules/ROOT/pages/bind-api-configure-service-mesh.adoc]: Good catch! Done! Thanks!

File: modules/ROOT/pages/manage-apis-with-service-mesh.adoc (hunk 7 above)
Comment (priyaayyar) [reply modules/ROOT/pages/manage-apis-with-service-mesh.adoc]: Done! Thanks!

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc (hunk 8 above)
Comment (priyaayyar) [reply modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: Done! Thanks!

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc (hunk 9 above)
Comment (priyaayyar) [reply modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: Done! Thanks!

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc (hunk 10 above)
Comment (priyaayyar) [reply modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: Done! Thanks!

**Review by tbruno — APPROVED** [review]
Thanks for addressing the feedback.

**Review by chiragnarangmulesoft — APPROVED** [review]

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc (hunk 10 above)
Comment (Mulesoft-Mysti) [reply modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: @kbostrom and @priyaayyar, Shouldn't this be an ordered list (numbers) instead of bullets since there is a specific order?

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc [hunk 11]
```diff
@@ -24,12 +29,14 @@ For more information about how to install Helm, see the Helm installation docume
...
 
-If you want to enable automatic sidecar injection, you must additionally have:
-* Install Kubernetes 1.12+ with admissionregistration API 
-*Tip:* To verify the installation, use the command `kubectl api-versions | grep admissionregistration.k8s.io/v1beta1`.
-* Configure kube-apiserver process with the admission-control flag to “MutatingAdmissionWebhook”
-For more information about how to turn on, see tke Kubernetes documentation.
-* Add and list ValidatingAdmissionWebhook admission controllers in the correct order.
+If you want to enable automatic sidecar injection, you must additionally:
+
+. Install Kubernetes 1.12+ with admissionregistration API.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: Should there be a "the" before admissionregistration?

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc [hunk 12]
```diff
@@ -24,12 +29,14 @@ For more information about how to install Helm, see the Helm installation docume
...
+. Verify the installation using the following the command: 
++
+`kubectl api-versions | grep admissionregistration.k8s.io/v1beta1`.
+. Configure kube-apiserver process with the admission-control flag to “MutatingAdmissionWebhook”.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: Configure the `kube-apiserver` process...

File: modules/ROOT/pages/bind-api-configure-service-mesh.adoc [hunk 13]
```diff
@@ -4,11 +4,9 @@ include::_attributes.adoc[]
...
+When you create the binding, Anypoint Service Mesh creates a three-way association of the service in Kubernetes, the API that you created in Anypoint Platform to represent the service, and the adapter that you created for the service. The adapter uses the unique API ID to fetch policies and apply them to your services that are running in the cluster.
 
-After you create the binding, Anypoint Service Mesh takes few seconds to complete the binding procedure and updates the status.
-
-After the binding is completed, you can view the status using the `list` command. The status values include:
+When you create the binding, Anypoint Service Mesh takes few seconds to complete the binding procedure and updates the status. After the binding is completed, you can view the status using the `list` command. The status values include:
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/bind-api-configure-service-mesh.adoc]: "updates" should be "update" (parallel with "complete" before it.) 
Subject: ASM
Predicate: takes
--- Infinitive: to complete the binding and update the status

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc (hunk 12 above)
Comment (priyaayyar) [reply modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: Done! Thanks!

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc (hunk 11 above)
Comment (priyaayyar) [reply modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: Done! Thanks!

File: modules/ROOT/pages/bind-api-configure-service-mesh.adoc (hunk 13 above)
Comment (priyaayyar) [reply modules/ROOT/pages/bind-api-configure-service-mesh.adoc]: Yes, you are right! Good catch!
